charts/ups-broker/templates/broker-deployment.yaml

```diff
@@ -27,6 +27,14 @@ spec:
         args:
         - --port
         - "8080"
+        {{- if .Values.tls.cert}}
+        - --tlsCert
+        - "{{ .Values.tls.cert }}"
+        {{- end}}
+        {{- if .Values.tls.key}}
```

Comment:
Both `{{- if .Values.tls.key }}` and `{{ if .Values.tls.key -}}` will yield the same yaml. The former will chomp the whitespace to the left, removing the newline from the previous line. The latter will chomp the whitespace to the right, removing the newline from the current line. In the end, they both will leave one newline. Using the left-side chomp seems to be a little less troublesome when the line after the if block is not at the same indentation level as the if block. With the right-side chomp, we would remove the spaces on the following line and leave the spaces on the current line, resulting in the following line being at the indentation of the current line.

For reference, the helm chart template guide [1] uses the left chomp in its example.

Here is the relevant snippet of the output from running `helm install charts/ups-broker --name ups-broker --namespace ups-broker --set tls.cert=fake-cert --set tls.key=fake-key --dry-run --debug`.
```
        imagePullPolicy: Always
        args:
        - --port
        - "8080"
        - --tlsCert
        - "fake-cert"
        - --tlsKey
        - "fake-key"
        ports:
        - containerPort: 8080
```

[1] https://github.com/kubernetes/helm/blob/master/docs/chart_template_guide/control_structures.md